src/dom/accessibility.ts

```diff
@@ -1,23 +1,40 @@
-// TODO: Before exposing this, it might be worth making `prefersReducedMotion`
-// a `MotionValue` with a selectively-exposed API, for instance `get` and `onChange`.
+import { motionValue } from "../value"
 
-let prefersReducedMotion: boolean | null = null
+interface ReducedMotionOptions {
+    /**
+     * Control whether to detect the device's Reduce Motion setting.
+     * @internal
+     */
+    detect: boolean
+
+    /**
+     * Whether to force reduce motion mode.
+     * @internal
+     */
+    force: boolean
```

Comment:
Is there a benefit to having these two decoupled? What happens if `force` is unset and `detect` is false? Do we fallback to a default value? 

It feels like passing force should just override the detection? What's the case where you'd want to disable detection but not force a setting?

Reply:
It's a good point. It is currently:

```
force={true} // Force reduce motion
force={false} // (default) Do not force reduce motion (detect)
```

It could be:

```
force={true} // Force reduce motion
force={false} // Force reduce motion off
force={null} // (default) Do not force reduce motion (detect)
```

Reply:
Yeah, I think that makes sense. I take it we need to specify that `null` or `undefined` will revert back to detection for use in a nested context?

Reply:
Do you think rather than detect and force we have a prop more explicit ie:

```
<ReduceMotion isReducedMotion={}>
```

Reply:
Or `enabled`

Reply:
I like `enabled`.